app/controllers/searches_controller.rb

```diff
@@ -3,11 +3,11 @@ class SearchesController < ApplicationController
 
   def show
     return unless params[:query]&.is_a?(String)
-    @error_msg, @gems = ElasticSearcher.new(params[:query], page: @page).search
+    @error_msg, @gems = ElasticSearcher.new(params[:query], page: @page, skip_exact_match: true).search
     limit_total_count if @gems.total_count > Gemcutter::SEARCH_MAX_PAGES * Rubygem.default_per_page
 
     @exact_match = Rubygem.name_is(params[:query]).with_versions.first
```

Comment:
IMHO, removing exact match feature/section altogether is a much simpler solution. A separate query was previously needed as we were using `LIKE` and there was no guarantee if an exact match would be shown at the top.
This is not an issue with elasticsearch, as far as I can tell the exact match results are always on the top/first.